src/main/resources/js/jstree/jstree.js

```diff
@@ -12,7 +12,7 @@
         // TODO: remove DEBUG lines
         console.log("jstree.js init");
         console.log("$ ver");
-        console.log($.fn.jquery);
+        //console.log($.fn.jquery); // $ is not found because we add <context>jira.view.issue</context> in the atlassian-plugin.xml
```

Comment:
This line and the line above should be just removed. The source code comment `// $ is not found ...` could be moved to the commit message.